### PR TITLE
fix: fixed XDAI e-mail notifications params

### PR DESCRIPTION
**Pull request by hbarcelos**

### CODE REVIEW

File: src/event-handlers/court-emails.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const qs = require('quertystring');
+const qs = require('querystring');
```
Comment (eccentricexit) [inline src/event-handlers/court-emails.js:1]: Ha! this one must've been annoying

File: src/event-handlers/court-emails.js [hunk 2]
```diff
@@ -193,20 +194,35 @@ const createLambdaHandler = (createWeb3, createContract) => {
...
+    let notifications = []
+    try {
+      notifications = await handlers[event.event](
+        web3,
+        createContract(web3),
+        event
+      );
+    } catch (err) {
+      const { event: eventName, ...payload } = event
+      console.info(`Error processing event ${event.event} with payload:`, payload)
```
Comment (eccentricexit) [inline src/event-handlers/court-emails.js]: Why not `console.error` here?

**Review by eccentricexit — CHANGES_REQUESTED** [review]

File: src/event-handlers/court-emails.js (hunk 1 above)
Comment (hbarcelos) [reply src/event-handlers/court-emails.js:1]: Yeah, it looks like my brain is kind of hard-wired to type `querty` whenever I want to type `query`. 😅

File: src/event-handlers/court-emails.js (hunk 2 above)
Comment (hbarcelos) [reply src/event-handlers/court-emails.js]: I probably copy-pasted the line and forgot to change the method.

**Review by eccentricexit — APPROVED** [review]
LGTM

**Pull request merged**